code/_global_vars/misc.dm

```diff
@@ -1,3 +1,5 @@
 GLOBAL_LIST_EMPTY(all_observable_events)
 
-GLOBAL_LIST_INIT(font_resources, list('fonts/Shage/Shage.ttf'))
\ No newline at end of file
+GLOBAL_LIST_INIT(font_resources, list('fonts/Shage/Shage.ttf'))
+
+GLOBAL_VAR_INIT(player_panel_id, "NewPlayerPanel")
```

Comment:
This var never changes; why not have it as a define?

Reply:
Yeah, no reason, just me thinking "we have a system for defining global variables, I should use it" without considering this is a constant. Will change tomorrow. ¬_¬

Reply:
Global var changed to define in misc.